cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs.get("row"),
+            attrs.get("seat"),
+            attrs.get("movie_session").cinema_hall,
             ValidationError
```

Comment:
In the `validate` method of `TicketSerializer`, the `validate_ticket` method is called with `ValidationError` as an argument. Ensure that `validate_ticket` is designed to accept this argument, or adjust the call accordingly. This could potentially lead to runtime errors if not handled properly.